### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by guilhermewaess**

### CODE REVIEW

File: task2/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "task2",
+  "version": "1.0.0",
+  "description": "Task 2 react mentoring",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run dev",
+    "build": "npm run prod",
+    "prod": "./node_modules/.bin/webpack --config ./build/webpack.prod.config.js",
+    "dev": "./node_modules/.bin/webpack-dev-server --config ./build/webpack.dev.config.js",
```
Comment (DenGogolin) [inline task2/package.json:10]: You can use npm installed module CLI without providing the relative or absolute path to the package.
E.g: 
"prod": "webpack --config ./build/webpack.prod.config.js",
"dev": "webpack-dev-server --config ./build/webpack.dev.config.js",

File: task2/build/webpack.base.config.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+const path = require('path');
+const webpack = require('webpack');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+const parentDir = path.join(__dirname, '../');
```
Comment (DenGogolin) [inline task2/build/webpack.base.config.js:5]: When you have webpack config in the folder other than the root one, it is good to use process.cwd() as an entry instead.

**Review by DenGogolin — COMMENTED** [review]
Everything is good. I left few suggestions.

File: task2/package.json (hunk 1 above)
Comment (guilhermewaess) [reply task2/package.json:10]: Sure, I changed! thanks.

File: task2/build/webpack.base.config.js (hunk 2 above)
Comment (guilhermewaess) [reply task2/build/webpack.base.config.js:5]: Make all sense. done :)

**Pull request merged**